nxpprog.py

```diff
@@ -398,6 +399,81 @@
 prog = None
 
 
+LPC_CHAR = {'Question': b'?',
+                'Synchronized': b'Synchronized\r\n',
```

Comment:
Can you fix the alignment of this section here.

Reply:
Done.